### PR TITLE
[apiv4] Add apiv4 for easy conversation api calls

### PR DESCRIPTION
**Pull request by nickvergessen**
- [x] Based on #1218

### CODE REVIEW

File: app/src/main/java/com/nextcloud/talk/utils/ApiUtils.java [hunk 1]
```diff
@@ -37,6 +37,8 @@
 import okhttp3.Credentials;
 
 public class ApiUtils {
+    public static final int APIv3 = 3;
+    public static final int APIv4 = 4;
```
Comment (nickvergessen) [inline app/src/main/java/com/nextcloud/talk/utils/ApiUtils.java:41]: Had to add configs to not get a warning about magic numbers (only -1 to 2 are okay, afterwards ktlint throws an error)

File: app/src/main/java/com/nextcloud/talk/activities/MainActivity.kt [hunk 2]
```diff
@@ -229,7 +229,7 @@ class MainActivity : BaseActivity(), ActionBarProvider {
         val roomType = "1"
         val currentUser = userUtils.currentUser ?: return
 
-        val apiVersion = ApiUtils.getConversationApiVersion(currentUser, intArrayOf(1))
+        val apiVersion = ApiUtils.getConversationApiVersion(currentUser, intArrayOf(ApiUtils.APIv4, 1))
```
Comment (AndyScherzinger) [inline app/src/main/java/com/nextcloud/talk/activities/MainActivity.kt:232]: for consistency probably also add `ApiUtils.APIv1` :)

**Review by AndyScherzinger — APPROVED** [review]
Just a minor thingy: cosmetics. Fien to be ignored ;)

File: app/src/main/java/com/nextcloud/talk/activities/MainActivity.kt (hunk 2 above)
Comment (nickvergessen) [reply app/src/main/java/com/nextcloud/talk/activities/MainActivity.kt:232]: let me do that once done, so my other PRs and branches don't break on the rebases even more :D

**Pull request merged**